mesonbuild/modules/rust.py

```diff
@@ -19,7 +19,8 @@
 from .. import mlog
 from ..build import BothLibraries, BuildTarget, CustomTargetIndex, Executable, ExtractedObjects, GeneratedList, IncludeDirs, CustomTarget, StructuredSources
 from ..dependencies import Dependency, ExternalLibrary
-from ..interpreter.interpreter import TEST_KWARGS, OUTPUT_KW
+from ..interpreter.interpreter import TEST_KWARGS, OUTPUT_KW, INCLUDE_DIRECTORIES
+from ..interpreter import type_checking
```

Comment:
INCLUDE_DIRECTORIES comes from the type_checking module, and should not be imported from an import. Admittedly, though, OUTPUT_KW is in the same boat and predates this PR.

... actually that one is my fault :rofl: introduced in e3d70d89b1ff17a34dd7e8d3de18647b26d171f0, shame on me.

Reply:
fixed. I moved `TEST_KWARGS` out of interpreter.py and into type_checking.py while I was at it, since that's probably what led to this problem. I sure wish that pylint or flake8 or mypy, or something had a check for implicitly rexporting...